### ISSUE TITLE
[NTR] Respiratory syncytial virus bronchiolitis

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#2559]
Preferred term label:
Respiratory syncytial virus bronchiolitis

Synonyms
RSV bronchiolitis

Definition (free text, please give PubMed ID)
Inflammation of the bronchioles, caused by the respiratory syncytial virus.
PMID: 28722988

Parent term (use hpo.jax.org/app)
Bronchiolitis

Related:
https://github.com/obophenotype/human-phenotype-ontology/issues/6660
